src/cockpit/beiboot.py

```diff
@@ -151,14 +152,19 @@ async def do_askpass(self, messages: str, prompt: str, hint: str) -> Optional[st
             # Let's avoid all of that by just showing nothing.
             return None
 
-        challenge = 'X-Conversation - ' + base64.b64encode(prompt.encode()).decode()
+        challenge_id = f'{os.getpid()}-{time.time()}'
```

Comment:
Can we make this a simple counter to make it easier to chat with this thing via `cockpit.print`?

Reply:
No, I tried that first. as beiboot gets invoked multiple times, and then always restarts with 0.

Reply:
Do we gain something by verifying the response, though?  We already matched this via the cookie...

Reply:
Not much, just additional confidence. It's mostly a safeguard in case the routing fails, to not take a non-matching response.